src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -0,0 +1,286 @@
+        dataStack.push(value.data.query);
+      } else if (value.type === DataType.DATA && dataStack.length === 2) {
+        dataStack.push(value.data);
+        setQueryContext({
+          question: dataStack[0],
+          query: dataStack[1],
+          queryResults: convertResult(dataStack[2]),
+        });
+        dataStack = [];
+      }
+    });
+
+    return () => {
+      combineSubscription.unsubscribe();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const fetchSummary = async () => {
+      if (isEmpty(queryContext?.queryResults)) return;
+      setLoading(true);
+      setSummary(null);
+      const SUCCESS_METRIC = 'fetch_summary_success';
+      try {
+        const actualSampleSize = Math.min(sampleSize, queryContext?.queryResults?.length);
+        const dataString = JSON.stringify(queryContext?.queryResults?.slice(0, actualSampleSize));
+        const payload = `'${dataString}'`;
+        const response = await props.http.post('/api/assistant/data2summary', {
+          body: JSON.stringify({
+            sample_data: payload,
+            sample_count: actualSampleSize,
+            total_count: queryContext?.queryResults?.length,
+            question: queryContext?.question,
+            ppl: queryContext?.query,
+          }),
+        });
+        setSummary(response);
+        reportCountMetric(SUCCESS_METRIC, 1);
+      } catch (error) {
+        reportCountMetric(SUCCESS_METRIC, 0);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchSummary();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [queryContext]);
+
+  useEffect(() => {
+    props.core.getStartServices().then(([coreStart, depsStart]) => {
+      const assistantEnabled = coreStart.application.capabilities?.assistant?.enabled === true;
+      setIsAssistantEnabledByCapability(assistantEnabled);
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const onFeedback = (satisfied: boolean) => {
+    if (feedback) return;
+    setFeedback(true);
+    reportMetric(satisfied ? 'thumbup' : 'thumbdown');
+  };
+
+  if (props.dependencies.isCollapsed || isQueryAssistCollapsed || !isAssistantEnabledByCapability)
+    return null;
+  return (
+    <EuiSplitPanel.Outer
+      className="queryAssist queryAssist__summary"
+      data-test-subj="queryAssist__summary"
+      hasBorder={true}
+      borderRadius="none"
+    >
+      <EuiSplitPanel.Inner
+        style={{ background: 'linear-gradient(to left, #EDF7FF, #F9F4FF)', padding: '6px' }}
+      >
+        <EuiFlexGroup alignItems={'center'} gutterSize={'xs'}>
+          <EuiFlexItem grow={false}>
+            <EuiIcon type={sparkleHollowSvg} size="m" />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiText size="s">
+              <strong>
+                {i18n.translate('assistantDashboards.vizSummary.response', {
```

Comment:
i18n library depends on the namespace path being accurate so it should be something like `queryEnhancements.queryAssistSummary...` (doesn't have to be exactly that but shouldn't start with assistantDashboards

Reply:
Thanks, I'll follow QueryAssistInput's way to name them